CompletedProcess(args=['python', 'temp_test_case.py'], returncode=1, stdout=b'', stderr=b'F.F......E\n======================================================================\nERROR: test9 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 66, in test9\n    self.assertEqual(patched_source(input_9), original_source(input_9))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_acc_qb.py", line 30, in patched_func\n    (a0, a1, a2, a3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 4)\n\n======================================================================\nFAIL: test0 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 12, in test0\n    self.assertEqual(patched_source(input_0), original_source(input_0))\nAssertionError: Lists differ: [\'TRIANGLE\'] != [None]\n\nFirst differing element 0:\n\'TRIANGLE\'\nNone\n\n- [\'TRIANGLE\']\n+ [None]\n\n======================================================================\nFAIL: test2 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 24, in test2\n    self.assertEqual(patched_source(input_2), original_source(input_2))\nAssertionError: Lists differ: [\'TRIANGLE\'] != [\'SEGMENT\']\n\nFirst differing element 0:\n\'TRIANGLE\'\n\'SEGMENT\'\n\n- [\'TRIANGLE\']\n+ [\'SEGMENT\']\n\n----------------------------------------------------------------------\nRan 10 tests in 0.001s\n\nFAILED (failures=2, errors=1)\n')